tutorials/Tutorial9_DPR_training.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>Did you check if we really just can fit batch_size=4 on a V100?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Yeah I did, I got an OOM Error with batch_size=8